gen3/bin/iam-serviceaccount.sh

```diff
@@ -422,7 +422,7 @@ function main() {
     if [ ${UPDATE_ACTION} == a ];
     then
       local addition_result=$(add_policy_to_role ${policy_source} ${role_name}) # ${policy_source})
-      if [ ${addition_result} == 0 ]; 
+      if [ ${addition_result} == 0 ];
       then
```

Comment:
let's delete this file and the docs if we want to use the awsrole stuff going forward ...